src/test/java/teammates/test/cases/webapi/GetCourseActionTest.java

```diff
@@ -97,15 +97,28 @@ protected void testAccessControl() throws Exception {
         };
 
         loginAsInstructor(instructor1OfCourse1.getGoogleId());
+
         verifyCannotAccess(submissionParams);
 
-        ______TS("only instructors of the same course can access");
+        ______TS("Inaccessible without being an instructor or student");
 
         submissionParams = new String[] {
                 Const.ParamsNames.COURSE_ID, courseAttributes.getId(),
         };
 
-        verifyOnlyInstructorsOfTheSameCourseCanAccess(submissionParams);
+        verifyInaccessibleWithoutLogin(submissionParams);
```

Comment:
You are advised to put this change in "Updated tests to support students" for a better practice of organizing commit messages

Reply:
Should it? I kind of disagree... this change is relevant to the change in access control for GetCourseAction, so the test should be updated in the same commit that contains that change?